### PR TITLE
Updating Getting Started (vnext)

### PR DESCRIPTION
**Pull request by harinikmsft**
Adding manual install steps and steps for running sample apps through cloning repo.

### CODE REVIEW

File: vnext/docs/GettingStarted.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Getting Started Guide for React Native Windows C++
+# Getting Started Guide for React Native Windows (vnext)
 
-This is a summary of setup steps needed to install and work with React Native for Windows10 (C++). See the [React Native Getting Started Guide](http://facebook.github.io/react-native/docs/getting-started.html) for React Native details.
+This is a summary of setup steps needed to install and work with React Native for Windows (vnext). See the [React Native Getting Started Guide](http://facebook.github.io/react-native/docs/getting-started.html) for React Native details and see [Getting Started Guide - current](https://github.com/microsoft/react-native-windows/blob/master/current/docs/GettingStarted.md) for working with the `current` React Native for Windows implementation.
 
 ## System requirements
 * You can run React-Native for Windows10 apps only on Windows 10 devices; sdk version: 10.0.14393.795 or higher.
```
Comment (licanhua) [inline vnext/docs/GettingStarted.md]: Here we mixed the running os and build target os. the correct description should be:
React Native Windows vNext only supports Windows version late than 10.0.16299.0, so it requires your project to have TargetPlatformMinVersion>=10.0.16299.0 when building.

Comment (harinikmsft) [reply vnext/docs/GettingStarted.md]: Good catch, I missed that instance. Updated now.

File: vnext/docs/GettingStarted.md [hunk 2]
```diff
@@ -52,30 +59,107 @@ If you are using Yarn, run
...
+
+* Run `Scripts\launchPackager.bat`.	This is needed to ensure the JS files can be packaged and bundled to the UWP app.
+
+* Make sure Chrome is running if you're not already running it
+
+#### Running the Playground app
+* Nuget restore through the command line. There is an outstanding issue [#2312](https://github.com/microsoft/react-native-windows/issues/2312) that blocks restoring Nuget dependencies using VS. 
+    ```cmd
+    cd Playground
+    ..\react-native-windows\vnext\Playground>nuget restore Playground.sln -PackagesDirectory Playground\packages
```
Comment (licanhua) [inline vnext/docs/GettingStarted.md]: I did a testing, and PackagesDirectory should be packages.
```suggestion
    ..\react-native-windows\vnext\Playground>nuget restore Playground.sln -PackagesDirectory packages
```

File: vnext/docs/GettingStarted.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
-# Getting Started Guide for React Native Windows C++
+# Getting Started Guide for React Native Windows (vnext)
 
-This is a summary of setup steps needed to install and work with React Native for Windows10 (C++). See the [React Native Getting Started Guide](http://facebook.github.io/react-native/docs/getting-started.html) for React Native details.
+This is a summary of setup steps needed to install and work with React Native for Windows (vnext). See the [React Native Getting Started Guide](http://facebook.github.io/react-native/docs/getting-started.html) for React Native details and see [Getting Started Guide - current](https://github.com/microsoft/react-native-windows/blob/master/current/docs/GettingStarted.md) for working with the `current` React Native for Windows implementation.
 
 ## System requirements
-* You can run React-Native for Windows10 apps only on Windows 10 devices; sdk version: 10.0.14393.795 or higher.
+* You can run React-Native for Windows10 apps only on Windows 10 devices; sdk version: 10.0.16299.0 or higher.
```
Comment (licanhua) [inline vnext/docs/GettingStarted.md]:
```suggestion
* You can run React-Native for Windows10 apps only on Windows 10 devices and Windows version: 10.0.16299.0 or higher.
```

**Pull request merged**